### ISSUE TITLE
Backups can't be listed when pimrary is not in ready state

### ISSUE THREAD

**Issue by saltenhub** [CrunchyData/postgres-operator#1927]
**Describe the bug**
Backups from a pgbackrest repository can't be listed while primary pod is not in ready state.

**To Reproduce**
1. Create cluster with local pgbackrest repository
2. Scale down primary node (in our case it crashed and could not be recovered)
3. ```pgo -n [ns] show backup [clustername]``` returns `Error: primary pod is not in Ready state`

**Expected behavior**
Backups can be listed, even if primary pod is not in ready state

**Please tell us about your environment:**

* Operating System: Centos
* Where is this running ( Local, Cloud Provider): Local
* Storage being used (NFS, Hostpath, Gluster, etc): Hostpath
* Container Image Tag: 4.4.1
* PostgreSQL Version: 12.1
* Platform (Docker, Kubernetes, OpenShift): K8S
* Platform Version: 1.17.9

As always: Thanks for great product and the support! :+1:

**Comment by jkatz** [comment]
@saltenhub Yeah, this is definitely interesting. The backup functionality typically runs the `pgbackrest` command from the primary instance, but I think it should be safe to run the `show` from the pgBackRest Pod. As a quick test:

```
kubectl -n pgo exec -it hippo-backrest-shared-repo-6d9cb8cc77-f9gbj -- pgbackrest info
stanza: db
    status: ok
    cipher: none

    db (current)
        wal archive min/max (12-1): 000000010000000000000001/00000002000000010000007C

        full backup: 20200927-150317F
            timestamp start/stop: 2020-09-27 15:03:17 / 2020-09-27 15:03:40
            wal start/stop: 000000010000000000000002 / 000000010000000000000002
            database size: 31.5MB, backup size: 31.5MB
            repository size: 3.8MB, repository backup size: 3.8MB

        incr backup: 20200927-150317F_20200928-191752I
            timestamp start/stop: 2020-09-28 19:17:52 / 2020-09-28 19:17:57
            wal start/stop: 00000002000000000000005E / 00000002000000000000005E
            database size: 31.5MB, backup size: 495.6KB
            repository size: 3.8MB, repository backup size: 77.8KB
            backup reference list: 20200927-150317F
```

So this seems like a reasonable thing we should make happen.

**Comment by IJOL** [comment]
But there isn't a way to make a replica act as new primary?

**Comment by jkatz** [comment]
This is fixed and backpatched all the way down to the 4.3.x series.

**Comment by IJOL** [comment]
Hi, how i can apply this fix to my 4.4.0 operator install? just pulling the images and restarting the pod should do the job?
